### PR TITLE
118 mailing list for event attendees

### PR DESCRIPTION
**Pull request by magikarp620**
Create button that return checked off attendees. 
Currently check condition with points != 0.

Could add more options including return RSVP and signed in attendees in the future if needed.

### CODE REVIEW

**Review by ryanyychen — COMMENTED** [review]
Hmm do you think maybe there's a more reliable way for checking for checked off? 
Lines 248-258 in the same file check for if the user has been checked off, I think maybe using a similar thing would be nice. Cause right now the rollover function actually sets their points to 0 (I know, this sucks, we will work on improving this).

**Comment by magikarp620** [comment]
Fixed.

File: frontend/src/Components/Events/CustomizableEventConsole.svelte [hunk 1]
```diff
@@ -84,6 +85,14 @@
             }
         });
 
+        emailsCheckedOff = [];
```
Comment (ryanyychen) [inline frontend/src/Components/Events/CustomizableEventConsole.svelte:88]: We already set emailsCheckedOff = [] in the very beginning (line 14). Might be a redundant line of code? Correct me if I'm wrong.

File: frontend/src/Components/Events/CustomizableEventConsole.svelte [hunk 2]
```diff
@@ -199,6 +222,7 @@
                     }}>
                     RSVP List
                 </button>
+                <button on:click={() => copyToClipboard(emailsCheckedOff)}>Copy Email</button>
```
Comment (ryanyychen) [inline frontend/src/Components/Events/CustomizableEventConsole.svelte:225]: Small issue, add an s to Email -> "Copy Emails".

**Review by ryanyychen — COMMENTED** [review]
Everything else looks good, just 2 more small things. Thanks.

File: frontend/src/Components/Events/CustomizableEventConsole.svelte (hunk 1 above)
Comment (magikarp620) [reply frontend/src/Components/Events/CustomizableEventConsole.svelte]: Initially, this is to make sure if the table regenerate the list starts empty so there are no duplicate emails. After testing without this line, the code still works so I deleted it.

File: frontend/src/Components/Events/CustomizableEventConsole.svelte (hunk 2 above)
Comment (magikarp620) [reply frontend/src/Components/Events/CustomizableEventConsole.svelte]: Fixed.

**Review by ryanyychen — APPROVED** [review]

**Pull request merged**